packages/MarkdownEditor-Core.package/MarkdownInlineParser.class/instance/nextTokenIn..st

```diff
@@ -0,0 +1,7 @@
+parsing
+nextTokenIn: aMarkdownReadStream
+
+	| tokenClass |
+	tokenClass := self tokenClasses detect: [:c | 
+		c beginsWith: aMarkdownReadStream peek asString].
```

Comment:
The reason are the upcoming `![` tokens. We need to somehow make sure, we can peek more than one char. Giving the stream to the `MarkdownToken` is a little counterintuitive, I think. Don't have a clue how this could be done nicely yet.